scripts/exglobal_prep_ocean_obs.py

```diff
@@ -1,17 +1,59 @@
 #!/usr/bin/env python3
 # exglobal_prep_ocean_obs.py
 # Pepares observations for marine DA
+from datetime import datetime, timedelta
+import logging
 import os
-from wxflow import YAMLFile
 import prep_marine_obs
-import logging
+import subprocess
+from wxflow import YAMLFile, save_as_yaml
+
+# set up logger
+logging.basicConfig(format='%(asctime)s:%(levelname)s:%(message)s', level=logging.INFO, datefmt='%Y-%m-%d %H:%M:%S')
+
+cyc = os.getenv('cyc')
+PDY = os.getenv('PDY')
+
+# TODO (AFE): ideally this should be an env var
+obsprocexec = "/scratch1/NCEPDEV/da/Andrew.Eichmann/fv3gfs/newoceaanobs/global-workflow/sorc/gdas.cd/build/bin/gdas_obsprovider2ioda.x"
+
+# set the window times
+cdateDatetime = datetime.strptime(PDY + cyc, '%Y%m%d%H')
+windowBeginDatetime = cdateDatetime - timedelta(hours=3)
+windowEndDatetime = cdateDatetime + timedelta(hours=3)
+windowBegin = windowBeginDatetime.strftime('%Y-%m-%dT%H:%M:%SZ')
+windowEnd = windowEndDatetime.strftime('%Y-%m-%dT%H:%M:%SZ')
 
 OBS_YAML = os.getenv('OBS_YAML')
+obsConfig = YAMLFile(OBS_YAML)
+print(obsConfig)
```

Comment:
use logger or remove.

Reply:
@guillaumevernieres One problem I;m having is getting logging to work. in the current version of the script I have:

```
logging.info('hippogriff')
print('jangle')
```

"jangle" shows up in the .out file and "hippogriff" does not. I can't figure out what I'm doing wrong here